Makefile

```diff
@@ -23,19 +23,22 @@ depends3:
 
 .PHONY: pipinstallpythonapt
 pipinstallpythonapt: pipinstallpythonapt_deps
-	$(MAKE) pipinstallpythonapt_default || $(MAKE) pipinstallpythonapt_src_$(UBUNTU_RELEASE)
+	$(MAKE) pipinstallpythonapt_src_$(UBUNTU_RELEASE)
```

Comment:
good point. That'll make building slighly longer because of the failure, but it makes sense to at least try it